### PR TITLE
seperate error messages during image processing

### PR DESCRIPTION
**Pull request by hkollmann**
during Image processing there are two checks: One for the Image file and one calculating the Image size.
To have better error Messages i seperated this two checks.

### CODE REVIEW

File: tool/pylib/generator/resource/Image.py [hunk 1]
```diff
@@ -331,7 +333,7 @@ def size(self):
         except (struct.error, IOError, KeyError):
             pass
         return (width, height)
-
+22
```
Comment (level420) [inline tool/pylib/generator/resource/Image.py]: I'm not really famialiar with python. What does this line mean?

Comment (hkollmann) [inline tool/pylib/generator/resource/Image.py]: Sorry, only a typo....

**Review by johnspackman — COMMENTED** [review]
LGTM but I dont know python and it appears that travis is not triggering automatic tests.  @hkollmann have you run unit tests for this change?  It would be great to have the unit tests extended to check for this new error condition

**Comment by level420** [comment]
@johnspackman I don't think we currently have unit tests which test generator.py itself. The testrunner is javascript only.

**Comment by hkollmann** [comment]
That's the problem. The Generator is'nt tested. Otherwise i had run the Tests :-)

**Comment by johnspackman** [comment]
aha, sorry I'd seen some toolchain tests (which test the output of the generator) and assumed that there were more...

**Review by level420 — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by cajus — APPROVED** [review]

**Comment by level420** [comment]
4 approvals, more than 2 business days
merging
thank you @hkollmann

**Pull request merged**